afs-storage-api/src/test/java/com/powsybl/afs/storage/AbstractAppStorageTest.java

```diff
@@ -558,6 +557,9 @@ public void onEvents(NodeEventList eventList) {
 
         NodeEvent eventToCatch = new NodeCreated("test2", "test");
         NodeEvent eventNotToCatch = new NodeCreated("test1", "test");
+
+        eventStack.clear();
```

Comment:
As it's asynchronous, is it ok to clear the stack? should we swallow the events (blocking call)?

Reply:
replaced by synchronous discard of events